advantage-disadvantage-javascript.md

```diff
@@ -0,0 +1,23 @@
+# Advantages & Disadvantages of JavaScript
+
+Like all computer languages, JavaScript has certain advantages and disadvantages. Many of the pros and cons are related to being a client-side language.
+
+### Advantages of JavaScript
+
+* Speed. Being client-side, JavaScript is very fast because any code functions can be run 	  immediately instead of having to contact the server and wait for an answer.
+
+* Simplicity. JavaScript is relatively simple to learn and implement.
+
+* Versatility. JavaScript plays nicely with other languages and can be used in a huge variety of applications. Unlike PHP or SSI scripts, JavaScript can be inserted into any web page regardless of the file extension. JavaScript can also be used inside scripts written in other languages such as Perl and PHP.
```

Comment:
I'm not familiar with SSI scripts. Could you provide a link to it, or maybe create an article about it and link this one to that one?